commit d7e508880b8f674aa82d3d5d9b2de27c034329a3
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 11:11:46 2010 -0700

    Staging: brcm80211: s/ushort/unsigned short/
    
    This changes the usage everywhere in the driver, and removes
    the definition as it should no longer be used anywhere.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/bcmutils.c
+++ b/drivers/staging/brcm80211/brcmfmac/bcmutils.c
@@ -807,16 +807,16 @@ int bcm_ether_atoe(char *p, struct ether
  * parameter order is like strncpy, but returns count
  * of bytes copied. Minimum bytes copied is null char(1)/wchar(2)
  */
-ulong wchar2ascii(char *abuf, ushort * wbuf, ushort wbuflen, ulong abuflen)
+ulong wchar2ascii(char *abuf, unsigned short * wbuf, unsigned short wbuflen, ulong abuflen)
 {
 	ulong copyct = 1;
-	ushort i;
+	unsigned short i;
 
 	if (abuflen == 0)
 		return 0;
 
 	/* wbuflen is in bytes */
-	wbuflen /= sizeof(ushort);
+	wbuflen /= sizeof(unsigned short);
 
 	for (i = 0; i < wbuflen; ++i) {
 		if (--abuflen == 0)
--- a/drivers/staging/brcm80211/include/bcmutils.h
+++ b/drivers/staging/brcm80211/include/bcmutils.h
@@ -212,7 +212,7 @@ extern "C" {
 	extern char *BCMROMFN(bcmstrcat) (char *dest, const char *src);
 	extern char *BCMROMFN(bcmstrncat) (char *dest, const char *src,
 					   uint size);
-	extern ulong wchar2ascii(char *abuf, ushort *wbuf, ushort wbuflen,
+	extern ulong wchar2ascii(char *abuf, unsigned short *wbuf, unsigned short wbuflen,
 				 ulong abuflen);
 	char *bcmstrtok(char **string, const char *delimiters, char *tokdelim);
 	int bcmstricmp(const char *s1, const char *s2);
--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -31,7 +31,6 @@ typedef unsigned long long int uintptr;
 #endif
 
 #define TYPEDEF_UINT
-#define TYPEDEF_USHORT
 #define TYPEDEF_ULONG
 
 /*
@@ -40,10 +39,6 @@ typedef unsigned long long int uintptr;
 
 /* define ushort, uint, ulong */
 
-#ifndef TYPEDEF_USHORT
-typedef unsigned short ushort;
-#endif
-
 #ifndef TYPEDEF_UINT
 typedef unsigned int uint;
 #endif
@@ -94,7 +89,6 @@ typedef signed int int32;
 
 #define	AUTO	(-1)		/* Auto = -1 */
 
-#undef TYPEDEF_USHORT
 #undef TYPEDEF_UINT
 #undef TYPEDEF_ULONG
 #undef TYPEDEF_UINT16
--- a/drivers/staging/brcm80211/include/wlioctl.h
+++ b/drivers/staging/brcm80211/include/wlioctl.h
@@ -566,8 +566,8 @@ typedef struct wl_wsec_key {
 
 /* receptacle for WLC_SET_WSEC_PMK parameter */
 typedef struct {
-	ushort key_len;		/* octets in key material */
-	ushort flags;		/* key handling qualification */
+	unsigned short key_len;		/* octets in key material */
+	unsigned short flags;		/* key handling qualification */
 	u8 key[WSEC_MAX_PSK_LEN];	/* PMK material */
 } wsec_pmk_t;
 
